src/host/main.cpp

```diff
@@ -187,6 +187,8 @@ int main(int argc, char** argv)
 
   LOG_INFO_FMT("CCF version: {}", ccf::ccf_version);
 
+  LOG_INFO_FMT("CLI args: {}", fmt::join(argv, argv + argc, " "));
```

Comment:
I don't know if there is a way to do that easily with fmt::something, but it would be nice to quote the args to avoid ambiguity if they contain spaces

This would log (I believe) `./a b c`, which can have been `./a "b c"` or `./a "b" "c"`.